src/components/accordion/Accordions.tsx

```diff
@@ -0,0 +1,93 @@
+import { FC, ReactNode } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import Typography from '@mui/material/Typography'
+import Box from '@mui/material/Box'
+import Accordion, { AccordionProps } from '@mui/material/Accordion'
+import AccordionSummary from '@mui/material/AccordionSummary'
+import AccordionDetails from '@mui/material/AccordionDetails'
+
+import { styles } from '~/components/accordion/Accordion.styles'
+
+import { AccordionItem, AccordionSx } from '~/types'
+import { Variant } from '@mui/material/styles/createTypography'
+
+interface AccordionsProps
+  extends Omit<AccordionProps, 'onChange' | 'children' | 'sx'> {
+  items: AccordionItem[]
+  onChange: (value: number) => void
+  activeIndex: number[] | number | null
+  multiple?: boolean
+  icon?: ReactNode
+  sx?: AccordionSx
+  titleVariant: Variant
+  descriptionVariant: Variant
+}
+
+const Accordions: FC<AccordionsProps> = ({
+  items,
+  onChange,
+  activeIndex,
+  icon,
+  sx = styles,
+  titleVariant,
+  descriptionVariant,
+  ...props
+}) => {
+  const { t } = useTranslation()
+
+  const isMultiple = Array.isArray(activeIndex)
+
+  const accordionStyle = sx[icon ? 'withIcon' : 'noIcon'] || {}
+
+  return (
+    <Box sx={accordionStyle.root}>
+      {items.map((item, index) => {
+        const active = isMultiple
```

Comment:
move map above and pass a variable in here, for better readability

Reply:
i'm not sure that it will increase readability, cause this map is one what we have inside of this component

Reply:
I think it will, return and map inside return doesn`t seem well